unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuiilderAPI/APIResponse/APIResponse.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using UnityEngine;
+
+namespace DCL.Builder
+{
+    public class APIResponse
+    {
+        public bool ok;
+        public string error;
+        public object data;
+
+        public string GetArrayJson()
```

Comment:
I will change the name of the function to GetArrayJsonString.

We can't have something like that because it is parsed directly from a typescript structure.  The array items is` ['data']['items]` which we lack a direct translation in C#. 

Most of the time the data objects contains just the data directly, but when it returns an array, they encapsulated them into an item category

Reply:
I have changed this to reflect it better, What do you think of the new approach? Will you like to change it?